graph/graph.go

```diff
@@ -112,11 +113,18 @@ func NewGraph(root string, driver graphdriver.Driver) (*Graph, error) {
 	}
 
 	graph := &Graph{
-		root:     abspath,
-		idIndex:  truncindex.NewTruncIndex([]string{}),
-		driver:   driver,
-		retained: &retainedLayers{layerHolders: make(map[string]map[string]struct{})},
+		root:             abspath,
+		idIndex:          truncindex.NewTruncIndex([]string{}),
+		driver:           driver,
+		retained:         &retainedLayers{layerHolders: make(map[string]map[string]struct{})},
+		tarSplitDisabled: false,
 	}
+
+	// Windows does not currently support tarsplit functionality.
+	if runtime.GOOS == "windows" {
+		graph.tarSplitDisabled = true
```

Comment:
On Windows, NewOutputTarStream will fail the checksum validation.  We definitely want to circle back on the investigation for why, but for the short term for TP3 it was much more straightforward to avoid the tarsplit so that we can fallback to normal diff behavior.